### DISCUSSION TITLE
앵그리버드처럼 조작하도록 시점을 고정시켜 개발할 수 있나요?

### DISCUSSION THREAD

**Discussion by timeisair** [naverz/zepeto-studio-kor#306] [4. General]
일반적인 제페토 시점이 아닌 앵그리버드처럼 횡화면으로 시점이 고정되고
앵그리버드 조작 방식으로 플레이하게 만드는것도 가능한지 궁금합니다. (_ _)

**Comment by runasy-koonta** [comment]
앵그리버드의 경우 2D인데, 2D 시점을 Zepeto에서 지원해주는지는 잘 모르겠네요.
시점 고정의 경우 가능합니다. #54 를 참고하시면 될 것 같아요.

**Reply by timeisair** [reply]
아... 지원 여부는 모르시는군요. 제페토에 물어봤었지만 매크로답변만 돌아오고.. 물어볼 곳이 없네요 ^^;
조언 고맙습니다 (_ _)

**Comment by mayko1234** [comment]
지원하진 않지만 직접 만드실 순 있어요. 저도 카메라 시점 고정해서 만들어봤어요.

**Reply by timeisair** [reply]
직접 만드는건 미니게임수준이지만 안드로이드로 출시를 했고 싱글솔로게임인데다 홍보부족으로 글자그대로 뭍혀버려서 
많이 아쉬웠었는데 제페토월드를 보고 멀티는 기본 지원이니 제 게임을 제페토월드로 출시하면 너무 좋겠단 생각을 했어요.

그래서 앵그라버드처럼 시점을 고정하는 것도 지원하는지가 궁금했었습니다. 
이미 프로젝트에 돌입하신 분들이면 아실 줄 알았거든요. ^^

아무래도 제페토 여느 게임들처럼 wasd로 조작하는 숄더 뷰 시점으로만 개발해야 하는 것 같군요.

조언 감사드리고 좋은 월드 개발하셔서 대박나시길 빕니다 ^^

**Reply by mayko1234** [reply]
아니예요 꼭 wsad로만 컨트롤 넣으실 필요없이 필요에 따라선 앵그리버드처럼 원키 조작법으로 만드는거 가능해요! 
(디폴트 캐릭터 컨트롤 사용하지 마시고, 제페토 스크립트로 인풋 조정하시면 되니까요~)
그리고 카메라도 측면에 두고 플레이어 따라가게 하거나, 
아니면 플레이어와 카메라를 고정시키고 오브젝트들이 다가오게 하는 식으로 구현하시면 되구요. 
감사합니다~ 대박나세요!

**Reply by timeisair** [reply]
그렇게해도 되는군요. 해봐야 알겠지만 일단 희망이 보이네요. 감사합니다 ^^

**Comment by cheolhui-lee** [comment]
안녕하세요.
크리에이터 분들의 편의를 위해 캐릭터와 연계된 디폴트 카메라를 제공하고는 있지만,
별도로 콘텐츠에 적합하게 직접 구현해서 사용하시는 것도 가능합니다.
참고로 Pongpong 이라는 게임도 카메라 제어는 별도로 구현을 한 게임이니 참고 부탁드립니다.

**Reply by timeisair** [reply]
점프마스터까지만 해봤었는데 말씀하신 게임을 직접 해보니 이해가 가네요. 고맙습니다!
